ProcessLib/GroundwaterFlowProcess.h

```diff
@@ -200,102 +177,65 @@ class GroundwaterFlowProcess : public Process
             _hydraulic_head->createNeumannBcs(
                     std::back_inserter(_neumann_bcs),
                     hydraulic_head_mesh_element_searcher,
-                    _global_setup,
+                    this->_global_setup,
                     _integration_order,
-                    *_local_to_global_index_map,
+                    *this->_local_to_global_index_map,
                     0,
                     *_mesh_subset_all_nodes);
         }
 
-        for (auto bc : _neumann_bcs)
-            bc->initialize(_global_setup, *_A, *_rhs, _mesh.getDimension());
-
+        Process<GlobalSetup>::initializeNeumannBcs(_neumann_bcs);
     }
 
-    void initialize() override
+    void initializeMeshSubsets(MeshLib::Mesh const& mesh) override
```

Comment:
Only a note: For now it is inconsistent to pass the mesh here, but to use the member in other places.